core/src/Streamly/Internal/Data/Stream/StreamD/Type.hs

```diff
@@ -108,6 +111,7 @@ import Streamly.Internal.Data.SVar.Type (State, adaptState, defState)
 import Streamly.Internal.Data.Unfold.Type (Unfold(..))
 
 import qualified Streamly.Internal.Data.Fold.Type as FL
+import qualified Streamly.Internal.Data.Parser.ParserD.Type as PD
```

Comment:
importing `as ParserD` would be clearer, we also use `PRD` at other places, if a smaller name is required then that should be preferred to be consistent.